packages/dx-scheduler-core/src/utils.test.ts

```diff
@@ -93,6 +94,48 @@ describe('Utils', () => {
         .toEqual(appointments.slice(0, 1));
     });
   });
+  describe('#excludedIntervals', () => {
+    const start = moment('2020-03-03');
+    it('should create an interval of excluded days', () => {
+      const excludedDays = [3];
+
+      expect(excludedIntervals(excludedDays, start))
+        .toEqual([[
+          moment(start.day(3)),
+          moment(start.day(3)).endOf('day'),
+        ]]);
+    });
+    it('should create several intervals of excluded days', () => {
+      const excludedDays = [2, 5];
+
+      expect(excludedIntervals(excludedDays, start))
+        .toEqual([[
+          moment(start.day(2)),
+          moment(start.day(2)).endOf('day'),
+        ], [
+          moment(start.day(5)),
+          moment(start.day(5)).endOf('day'),
+        ]]);
+    });
+    it('should create an interval for several excluded days in a row', () => {
+      const excludedDays = [2, 3, 4, 5];
+
+      expect(excludedIntervals(excludedDays, start))
+        .toEqual([[
+          moment(start.day(2)),
+          moment(start.day(5)).endOf('day'),
+        ]]);
+    });
+    it('should work correctly with sundays', () => {
```

Comment:
What difference between `sundays` and basic days?

Maybe call this test something like that - `should work with week boundaries`